pkgs/applications/radio/direwolf/default.nix

```diff
@@ -22,22 +22,23 @@ stdenv.mkDerivation rec {
     espeak gpsd hamlib perl python3
   ] ++ (optionals stdenv.isLinux [alsa-lib udev]);
 
-  patches = [
-    ./udev-fix.patch
-  ];
-
   postPatch = ''
+    substituteInPlace conf/CMakeLists.txt \
+      --replace /etc/udev/rules.d/ $out/lib/udev/rules.d/
     substituteInPlace src/symbols.c \
       --replace /usr/share/direwolf/symbols-new.txt $out/share/direwolf/symbols-new.txt \
       --replace /opt/local/share/direwolf/symbols-new.txt $out/share/direwolf/symbols-new.txt
     substituteInPlace src/decode_aprs.c \
       --replace /usr/share/direwolf/tocalls.txt $out/share/direwolf/tocalls.txt \
       --replace /opt/local/share/direwolf/tocalls.txt $out/share/direwolf/tocalls.txt
     substituteInPlace scripts/dwespeak.sh \
+      --replace /bin/bash /bin/sh \
```

Comment:
```suggestion
      --replace /bin/bash ${bash}/bin/bash \
```

Reply:
I just updated this to use `patchShebangs`, which I forgot was the canonical way to do this.